pkg/ocicni/ocicni.go

```diff
@@ -278,13 +278,13 @@ func loadNetworks(confDir string, cni *libcni.CNIConfig) (map[string]*cniNetwork
 		if strings.HasSuffix(confFile, ".conflist") {
 			confList, err = libcni.ConfListFromFile(confFile)
 			if err != nil {
-				logrus.Warningf("Error loading CNI config list file %s: %v", confFile, err)
+				logrus.Errorf("Error loading CNI config list file %s: %v", confFile, err)
```

Comment:
I'm slightly new to this code base, so I'd like to understand what's the advantage of switching the "Warning" to "Error", considering this is not a fatal issue.

Reply:
fixed!

Reply:
Hey @wgahnagl , you mentioned it was fixed but it seems that it's not.  The diff still shows the change switching from "Warning"  to "Error".